tasks_examples/OALL_tasks_test.txt

```diff
@@ -0,0 +1,4 @@
+lighteval|arabic_mmlu:abstract_algebra|5|1
```

Comment:
I don't think this file is needed :)

Reply:
Yah it was just for test and totally forgot about it and got pushed as well